cmd/lncli/commands.go

```diff
@@ -2324,6 +2329,14 @@ func exportChanBackup(ctx *cli.Context) error {
 			Index: chanPointRPC.OutputIndex,
 		}
 
+		if outputFileName != "" {
+			return ioutil.WriteFile(
```

Comment:
While we're touching this code, we should also change the deprecated `ioutil.WriteFile` to the new `os.WriteFile`, here and below.

Reply:
Agreed!